syntax/verilog_systemverilog.vim

```diff
@@ -312,6 +312,14 @@ if index(s:verilog_syntax_fold, "define") >= 0 || index(s:verilog_syntax_fold, "
     " verilogGlobal keywords. This syntax match fixes that:
     syn match verilogGlobal "\<`\(ifn\?def\|e\(els\(e\|if\)\)\|ndif\)\>"
 endif
+if index(s:verilog_syntax_fold, "instance") >= 0 || index(s:verilog_syntax_fold, "all") >= 0
+    syn region verilogFold
+        \ start="^\s*\w\+\(\s*#\s*(\|\s\+\w\+\s*(\)"
```

Comment:
I think that that would only work if the module was instanced as `module_name #(....) instance_name`, which is not very common. But my latest version does not allow this type of instance, so it isn't better :(